core/src/main/java/com/amplifyframework/core/model/query/QueryPaginationInput.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.core.model.query;
+
+import androidx.annotation.NonNull;
+
+import java.util.Objects;
+
+/**
+ * A simple data structure that holds pagination information that can be applied queries.
+ */
+public class QueryPaginationInput {
+
+    /**
+     * The default page size.
+     */
+    public static final Integer DEFAULT_LIMIT = 100;
```

Comment:
Can we use the term "page size", instead of limit? "limit" is ambiguous. It would either be a max results, or a number of results per page. I guess here it means limit for each page.